lib/collective.lib.js

```diff
@@ -157,9 +158,21 @@ export const getSuggestedTags = collective => {
 
 /** Checks if recurring contributions are allowed for the user for a given collective **/
 export const canContributeRecurring = (collective, user) => {
-  return (
-    collective.host.supportedPaymentMethods.some(paymentMethod =>
-      ProvidersWithRecurringPaymentSupport.includes(paymentMethod),
-    ) || user?.memberOf.some(member => member.collective?.host?.id === collective.host.legacyId)
+  // If the host has a a payment method that supports recurring payments (PayPal, Credit Card, etc.)
```

Comment:
@Betree : Everything looks good, except for this minor typo `a a payment method`. 😉